package.json

```diff
@@ -42,7 +42,7 @@
     "cbq": "git+https://github.com/bitfinexcom/cbq-js.git",
     "eosjs": "^20.0.0",
     "isomorphic-ws": "^4.0.1",
-    "mandelbrot": "https://github.com/bitfinexcom/mandelbrot.git#4a810a3caf4f1e89e563c68d8ee43dbbeb1c0411",
+    "mandelbrot": "github:bitfinexcom/mandelbrot#master",
```

Comment:
Wouldn't it be better to use specific commit or create tags in case mandelbrot and sunbeam need to be updated together?
For example, you may update mandelbrot first, which may break sunbeam.

Reply:
I don't have permission to create tags on the repository.  I think pointing to master on `mainnet` is ok for now since `mandelbrot` is not likely to change much going forward.  If we do start to change it a lot a tag or a commit hash would be better I agree.